.metamaskrc.dist

```diff
@@ -4,6 +4,10 @@
 ; This variable is required
 INFURA_PROJECT_ID=00000000000
 
+; This variable is not required but it's necessary for the storybook
+; to render stories that use onchain data.
+INFURA_STORYBOOK_PROJECT_ID=
```

Comment:
This is the new ENV introduced to handle a dedicated Infura ID only for Storybook builds

Reply:
We may need to discuss with someone on how we can populate this into our CI system.

Reply:
@Prithpal-Sooriya I added the variable in CircleCI

<img width="239" alt="Screenshot 2024-10-25 at 12 13 10" src="https://github.com/user-attachments/assets/b463a48c-77ff-4578-b1c6-599c15343d47">